src/components/Comments/CommentForm.tsx

```diff
@@ -25,8 +26,10 @@ export const CommentForm: FC<CommentFormProps> = observer(({
   maxRows = 4,
 }) => {
   const formRef = useRef<HTMLFormElement>(null);
-  const actionRef = useRef<{ update?: (text?: string) => void }>({});
-
+  const actionRef = useRef<{ update?: (text?: string) => void, ele?: RefObject<HTMLTextAreaElement> }>({});
```

Comment:
```suggestion
  const actionRef = useRef<{ update?: (text?: string) => void, el?: RefObject<HTMLTextAreaElement> }>({});
```
It is really nitpicky but conventional shortened form of element is `el`, can we change that?